projects/ng-wizard/src/lib/ng-wizard.service.ts

```diff
@@ -132,9 +133,30 @@ export class NgWizardService {
   /**
    * Returns the Angular Route for the wizard component found in Angular's router config.
    * @param wizardComponentName The name of the wizard component
+   * @param wizardName The unique name of the wizard
    */
-  private getWizardRoute(wizardComponentName: string): Route {
-    return this.router.config.find((route) => route.component && route.component.name === wizardComponentName);
+  private getWizardRoute(wizardComponentName: string, wizardName: string): Route {
+    const wizardRoutes = this.getAllWizardRoutes(this.router.config, wizardComponentName);
+    if (wizardName) {
+      return wizardRoutes.find((route) => route.data && route.data.name === wizardName);
+    } else {
+      return wizardRoutes.find((route) => !route.data || !route.data.name);
```

Comment:
just a cosmetic change: the value being returned does not need to be in the 'else'